app/src/androidTest/java/com/plcoding/calorytracker/TrackerOverviewE2E.kt

```diff
@@ -87,7 +88,7 @@ class TrackerOverviewE2E {
             trackerUseCases = trackerUseCases,
             filterOutDigits = FilterOutDigits()
         )
-        composeRule.setContent {
+        composeRule.activity.setContent {
```

Comment:
I fixed because of this issue.

https://github.com/philipplackner/CalorieTracker/issues/5